### PR TITLE
chore: remove unnecessary props from button and improve test color co…

### LINKED ISSUE #22286: Remove redundant props (isActive and isHover) from headless and wds button
Delete the following props
- isFocused
- isActive
- isHover

### PR DESCRIPTION
**Pull request by KelvinOm**
## Description
Removed redundant props from the button and improved the component for testing the color generation algorithm

Fixes #22286

Media
Снимок экрана 2023-05-03 в 18 34 20

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### CODE REVIEW

**Review by ichik — DISMISSED** [review]

**Review by ichik — APPROVED** [review]

File: app/client/packages/design-system/widgets/src/components/Button/index.styled.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const StyledButton = styled(HeadlessButton)<ButtonProps>`
   outline: 0;
   gap: var(--spacing-4);
   padding: var(--spacing-2) var(--spacing-4);
-  min-height: calc(var(--sizing-root-unit) * 8);
+  height: calc(var(--sizing-root-unit) * 8);
```
[the remark this answers is not in the document]
Comment (KelvinOm) [reply app/client/packages/design-system/widgets/src/components/Button/index.styled.tsx:13]: There is a separate `isFitContainer` prop for setting the height and width of the container. In this case, we do not need the height to stretch across the container.

**Review by jsartisan — APPROVED** [review]

**Pull request merged**